### DISCUSSION TITLE
EQ and audio filter

### DISCUSSION THREAD

**Discussion by OneBitterBit** [cpyarger/obs-midi#107] [Ideas]
First I'd like to thank you for creating this plugin, it's wonderful and works great!
It occurred to me that it would be very useful if you could act on the parameters of the audio filters such as the equalization, the source of the right / left stereo sound and maybe on vst plugins such as the suite http://reaper.fm/reaplugs /.
In any case, I thank you immensely for the work already done! If you have something where you can donate, I gladly do it in my small way to support you!

**Comment by cpyarger** [comment]
Controlling filters is definitely on the list, It is a bit more complex than the basic features we are still fleshing out. so has been put on the back burner for the moment as it will be a significant undertaking. 

I am not certain if I will be able to control VSTs as they open an entirely different can of worms. 

Any donations you can do by clicking the sponsor link --^ up top. Not required, but much appreciated if you decide to!
